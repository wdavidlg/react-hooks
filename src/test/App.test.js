import {shallow} from 'enzyme'
import App from '../App';

describe('Pruebas con <App/>', () => {
    test('debe mostrarse correctamente', () => {
        const wrapper = shallow(<App/>);
        expect(wrapper).toMatchSnapshot();
    })
    
})
