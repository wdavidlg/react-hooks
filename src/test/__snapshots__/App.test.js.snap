// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas con <App/> debe mostrarse correctamente 1`] = `
<h1>
  Hook App
</h1>
`;
